frontend/src/components/pages/CourseOverview.tsx

```diff
@@ -1,25 +1,67 @@
 import React from "react";
-import { Box, Flex, VStack } from "@chakra-ui/react";
+import { useHistory, useParams } from "react-router-dom";
+import { ChevronLeftIcon } from "@chakra-ui/icons";
+import { Box, Button, Flex, Text, VStack } from "@chakra-ui/react";
+import { useQuery } from "@apollo/client";
 import Banner from "../learner/Banner";
 import ModuleLessonCount from "../learner/ModuleLessonCount";
+import { GET_COURSE } from "../../APIClients/queries/CourseQueries";
+import { COURSE_OVERVIEW_BASE_ROUTE } from "../../constants/Routes";
+import { CourseOverviewParams } from "../../types/CourseOverviewTypes";
 
 const ViewCourse = (): React.ReactElement => {
+  const history = useHistory();
+  const { courseID }: CourseOverviewParams = useParams();
+  const { data: courseData } = useQuery(GET_COURSE, {
+    variables: {
+      id: courseID,
+    },
+  });
+
+  const onReturnToCourseOverviewClick = () => {
+    history.push(COURSE_OVERVIEW_BASE_ROUTE);
+  };
+
+  const getLessonCount = () => {
+    let lessonCount = 0;
+
+    courseData?.course?.modules?.forEach((moduleContent: any) => {
+      lessonCount += moduleContent?.lessons?.length;
+    });
+
+    return lessonCount;
+  };
+
   return (
     <Flex direction="column">
       <Banner />
       <Flex
         px="120px"
         py="20px"
         height="400px"
-        background="background.dark"
+        background="background.lightgrey"
         color="white"
         align="center"
       >
         <VStack flex="2" align="start">
-          <span>breadcrumb here</span>
-          <span>title here</span>
-          <span>description here</span>
-          <ModuleLessonCount color="white" moduleCount={10} lessonCount={24} />
+          <Box display="flex">
+            <Button onClick={onReturnToCourseOverviewClick} variant="link">
+              <ChevronLeftIcon color="brand.royal" mr="15px" boxSize={30} />
+              <Text variant="display-xs" color="brand.royal">
+                Return to Course browsing
```

Comment:
I think it's a typo in the particular screenshot we gave you, but most of the designs have the "B" in "Browsing" capitalized. Would you mind updating it?

The one we gave you:
![image](https://user-images.githubusercontent.com/9922514/182772034-763eef67-d4a2-4182-8947-f839791d92d3.png)
The other ones:
![image](https://user-images.githubusercontent.com/9922514/182772006-49bfb770-4b15-4719-922e-1eea6f4f1c94.png)